.changeset/rare-parents-pretend.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/create-app': patch
+---
+
+Use `ServerPermissionClient` instead of `PermissionAuthorizer`.
```

Comment:
yeah I was thinking the same.
Since `ServerPermissionClient` is more specific, it could make things easier to follow, compared to a dummy interface. But at the same time having the interface might give more flexibility.
Also, I'm noticing that all the plugins' `RouterOptions` are expecting `PermissionEvaluator`, so probably that's the way to go.

Reply:
done